projects/cds/src/merchandising/cms-components/merchandising-carousel/merchandising-carousel.component.service.ts

```diff
@@ -159,6 +162,22 @@ export class MerchandisingCarouselComponentService {
       : [EMPTY];
   }
 
+  private mapStrategyProductsToProductIds(
+    strategyProducts: StrategyProducts
+  ): string[] {
+    return strategyProducts && strategyProducts.products
+      ? strategyProducts.products.map(strategyProduct=>strategyProduct.id)
+      : [];
+  }
+
+  private getMerchandisingCarouselModelId(
+    cmsComponent: CmsMerchandisingCarouselComponent,
+    productsIds: string[]
+  ): string {
+    const products = productsIds.reduce((acc,pId)=>acc+"_"+pId,"");
+    return  cmsComponent.uid + "_" + cmsComponent.strategy + products;
```

Comment:
This code doesn't seem to be formatted according to prettier. Please format it in order to make the CI pass.

Reply:
Should be fixed